### PR TITLE
br: public parameter --ignore-stats to backup statistic data

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add description about `--ignore-stats`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/15305
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-manual.md [hunk 1]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
 ADMIN RELOAD BINDINGS;
 ```
 
+## Restore statistics
+
+Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. It supports backing up and restoring statistics of column, index, and table levels. After you enable this feature, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:259]:
```suggestion
Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of column, index, and table levels. You do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```

File: br/br-snapshot-manual.md [hunk 2]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
...
+
+This parameter is enabled by default, and the configuration item is `--ignore-stats=true`, which means that table statistics are not backed up when the data is backed up.
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:261]:
```suggestion
If you do not set this parameter to `false`, the `br` command-line tool uses the default setting `--ignore-stats=true`, which means statistics are not backed up during data backup.
```

File: br/br-snapshot-manual.md [hunk 3]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
...
+
+The following is an example of backing up cluster snapshot data and backing up table statistics with `--ignore-stats=false`:
+
+```shell
+br backup full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
+--ignore-stats=false
+```
+
+After completing the above configuration, when you restore data, if the table statistics are included in the backup set, the `br` command line tool automatically restores the table statistics during restoring data:
```
Comment (hfxsd) [inline br/br-snapshot-manual.md:271]:
```suggestion
After completing the above configuration, when you restore data, if the table statistics are included in the backup, the `br` command line tool automatically restores the table statistics during restoring data:
```

File: br/br-snapshot-manual.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ This document describes the commands of TiDB snapshot backup and restore accordi
     - [Restore a table](#restore-a-table)
     - [Restore multiple tables with table filter](#restore-multiple-tables-with-table-filter)
     - [Restore execution plan bindings from the `mysql` schema](#restore-execution-plan-bindings-from-the-mysql-schema)
+- [Restore statistics](#restore-statistics)
```
Comment (qiancai) [inline br/br-snapshot-manual.md:22]: 中文里修改位置在
image

英文需改在
image

因为英文翻译也需要是 Back up statistics

File: br/br-snapshot-manual.md [hunk 5]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
 ADMIN RELOAD BINDINGS;
 ```
 
+## Restore statistics
```
Comment (qiancai) [inline br/br-snapshot-manual.md:257]: 同上，位置和名字

File: br/br-snapshot-manual.md [hunk 6]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
...
+
+Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of column, index, and table levels. In this case, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```
Comment (qiancai) [inline br/br-snapshot-manual.md:259]:
```suggestion
Starting from TiDB v7.5.0, the `br` command-line tool introduces the `--ignore-stats` parameter. When you set this parameter to `false', the `br` command-line tool supports backing up and restoring statistics of columns, indexes, and tables. In this case, you do not need to manually run the statistics collection task for the TiDB database restored from the backup, or wait for the completion of the automatic collection task. This feature simplifies the database maintenance work and improves the query performance.
```

File: br/br-snapshot-manual.md [hunk 7]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
...
+
+The following is an example of backing up cluster snapshot data and backing up table statistics with `--ignore-stats=false`:
+
+```shell
+br backup full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log \
+--ignore-stats=false
+```
+
+After completing the above configuration, when you restore data, if the table statistics are included in the backup, the `br` command-line tool automatically restores the table statistics during restoring data:
```
Comment (qiancai) [inline br/br-snapshot-manual.md:271]:
```suggestion
After backing up data with the preceding configuration, when you restore data, the `br` command-line tool automatically restores table statistics if table statistics are included in the backup:
```

File: br/br-snapshot-manual.md [hunk 8]
```diff
@@ -253,6 +254,29 @@ DELETE FROM bind_info WHERE original_sql = 'builtin_pseudo_sql_for_bind_lock' LI
...
+
+```shell
+br restore full \
+--storage local:///br_data/ --pd "${PD_IP}:2379" --log-file restore.log
+```
+
+When the backup and restore feature backs up data, it stores the statistics in JSON format in the `backupmeta` file. When restoring data, it imports the statistics in JSON format into the cluster. For more information, see [LOAD STATS](/sql-statements/sql-statement-load-stats.md).
```
Comment (qiancai) [inline br/br-snapshot-manual.md:278]:
```suggestion
When the backup and restore feature backs up data, it stores statistics in JSON format within the `backupmeta` file. When restoring data, it loads statistics in JSON format into the cluster. For more information, see [LOAD STATS](/sql-statements/sql-statement-load-stats.md).
```

File: br/br-snapshot-manual.md (hunk 5 above)
Comment (hfxsd) [reply br/br-snapshot-manual.md]: fixed

File: br/br-snapshot-manual.md (hunk 4 above)
Comment (hfxsd) [reply br/br-snapshot-manual.md]: fixed

**Review by Leavrth — APPROVED** [review]

**Review by benmaoer — APPROVED** [review]
LGTM

**Pull request merged**